git_repo/services/service.py

```diff
@@ -156,6 +156,7 @@ def __init__(self, r=None, c=None):
                                                       c.get('privatekey', None))))
         self._alias = c.get('alias', self.name)
         self.fqdn = c.get('fqdn', self.fqdn)
+        self.insecure = c.get('insecure', False)
```

Comment:
This doesn't really work, since self.insecure is stored as a string which never evaluates to False, even if it contains the string `'false'`.